server/index.js

```diff
@@ -26,7 +26,7 @@ const cohortsRouter = require('./routes/cohort');
 const listsRouter = require('./routes/list');
 const mailerRouter = require('./routes/mailer');
 const activitiesRouter = require('./routes/activity');
-const unlockLocks = require('./middleware/cleanLocks');
+const unlockLocks = require('./common/utils/cleanLocks');
```

Comment:
Consider using a consistent name with the default export name ("unlockLocks.js").